scripts/core/editor3/html/to-html/plainTextToHtml.ts

```diff
@@ -0,0 +1,6 @@
+// converts linebreaks in paragraphs
+export const plainTextToHtml = (text: string) =>
+    text.replace(/.*[^\n]/g, '<p>$&</p>');
```

Comment:
how about ```str.split('\n').map((line) => `<p>${line}</p>`).join('')```? Is your regex doing something different?

Reply:
Nope, it's just doing that

Reply:
Can you use the JS version then so I don't need a regex manual to understand it? :D